qa/remote-clusters/src/test/java/org/opensearch/cluster/remote/test/RemoteClustersIT.java

```diff
@@ -123,10 +125,11 @@ public void testHAProxyModeConnectionWorks() throws IOException {
 
         RemoteConnectionInfo rci = cluster1Client().cluster().remoteInfo(new RemoteInfoRequest(), RequestOptions.DEFAULT).getInfos().get(0);
```

Comment:
Shouldn't this whole block 126-131 be inside of the `assertBusy(...)` block so it makes a new network request to retrieve the connection info?

Reply:
Thanks, you're right. Somehow i missed it. I've updated the test now.